docs/source/getting-started.rst

```diff
@@ -0,0 +1,90 @@
+
+===============
+Getting started
+===============
+
+This getting started guide walks you through using Anaconda Project for the first time. 
+
+**After completing this guide, you will be able to:**
+
+* Create a project containing a Bokeh app
+* Package and share the project
+* Run the project with a single command
+
+This guide is for all platforms: Windows, OS X and Linux.
+
+If you have not yet installed and started Anaconda Project, follow the :doc:`Install instructions <install>`.
+
+NOTE: Windows users using Python 3.5.1 should upgrade to Python 3.5.2+ with the command ``conda update python`` to avoid an issue with Windows and Python 3.5.1.
```

Comment:
Can we please move this note to the location in the process where they need to do it? If this is the right location - after installing Anaconda Project, then it should read as follows:

WINDOWS USERS: If you use Python 3.5.1, next upgrade to Python 3.5.2+ with the command ``conda update python`` to avoid a known issue with Windows and Python 3.5.1.